tools/uninstall.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+run_with_echo() {
+  echo "$@" && eval "$@" || exit $?
+}
+
+run_with_echo sudo rm -rf "/Library/Input Methods/Gureum.app"
```

Comment:
`run_with_echo`의 한계로 따옴표를 이중으로 감싸줘야 합니다.
```suggestion
run_with_echo sudo rm -rf "\"/Library/Input Methods/Gureum.app\""
```

테스트로 다음 스크립트를 실행해보면 알 수 있습니다.
``` sh
$ run_with_echo ls "/Library/Input Methods/Gureum.app"
ls /Library/Input Methods/Gureum.app
ls: /Library/Input: No such file or directory
ls: Methods/Gureum.app: No such file or directory
```
따옴표를 붙여주면 `echo`와 `eval`에도 따옴표가 들어가서 제대로 실행됩니다.
``` sh
$ run_with_echo ls "/Library/Input Methods/Gureum.app"
ls "/Library/Input Methods/Gureum.app"
Contents
```